src/features/Tiles/BigMovieTile/styled.js

```diff
@@ -0,0 +1,159 @@
+import styled from "styled-components";
+
+export const MovieDetailsTile = styled.article`
+    padding: 40px;
+    background: ${({ theme }) => theme.colors.tile.background};
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    margin: 40px 8px 64px;
```

Comment:
please remove margin

Reply:
Removed, but probably only temporarily because there must be a distance from other elements anyway.